omod/src/main/java/org/openmrs/web/controller/user/UserFormController.java

```diff
@@ -9,20 +9,9 @@
  */
 package org.openmrs.web.controller.user;
```

Comment:
Did you make any real changes to this file? If not, just revert these code format changes.

Reply:
Yes we would like to set the HTTP response code to 400 when error occurs https://github.com/openmrs/openmrs-module-legacyui/pull/113/files#diff-03601c5bd9c5657d7a3bdcf55cd2f3d3R282